graphics/openvdb/Portfile

```diff
@@ -7,8 +7,7 @@ PortGroup               active_variants 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               boost 1.0
 
-github.setup            AcademySoftwareFoundation openvdb 8.0.0 v
-revision                1
```

Comment:
please leave the `revision` field but reset it to `0`

Reply:
Done